pints/_transformation.py

```diff
@@ -708,13 +708,13 @@ def to_search(self, p):
 class RectangularBoundariesTransformation(Transformation):
     r"""
     A generalised version of the logit transformation for the model parameters,
-    which transform an interval or rectangular boundaries :math:`[a, b)` to
+    which transform an interval or rectangular boundaries :math:`(a, b)` to
```

Comment:
transform -> transforms.

Is it a () interval then, not a half-open one?

Reply:
() is an interval, yes.

Reply:
But you're saying the interval doesn't include `a` then? So it's an open rather than a half-open interval?

Reply:
Ha, need correcting that line too! :smile:

Reply:
It's an open () interval. I think now the docs is consistent...?

Reply:
:D Are you sure? The rectangular boundaries docs say they define a half-open interval https://pints.readthedocs.io/en/stable/boundaries.html#pints.RectangularBoundaries

Reply:
Oh... didn't realise that!

Reply:
I can't remember the reason - why do we want to include the lower bound?

Reply:
Not sure, but maybe a separate ticket then? This can just fix the missing "s" and test :D :D :D

Reply:
See #1279.